brainstormforce/ultimate-addons-for-gutenberg
# Fix: WP Search - Hide Typography In case of button type is icon.

### Description

- WP Search > Hide Typography In case of button type is icon-  https://share.bsf.io/Z4ujgLke


### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?

- Drag and drop WP Search.
- Select Text and check typo is visible or not.
- Select Icon and typo should be hde.

### Checklist:
